src/controller/oscqueryBridgeController.ts

```diff
@@ -213,17 +210,22 @@ export class OSCQueryBridgeControllerPrivate {
 	}
 
 	private async _initAudio(state: OSCQueryRNBOState) {
+		console.log("init audio");
```

Comment:
do you want to keep this in here?

Reply:
https://github.com/Cycling74/rnbo-runner-panel/pull/203/commits/1fbdf03fd433709a4420a847dd9e4d66228b4770